src/pool.ts

```diff
@@ -100,7 +100,7 @@ export class LLMInstance {
 	}
 	chatStateHash?: string
 	fingerprint: string
-
+        created_by: Date
```

Comment:
should be camel case. theres also a `createdAt` in the pools createChatCompletion. can call it the same here.

Reply:
in the api reference, it is `created`, so i just commited and changed that.